python/ribasim/ribasim/config.py

```diff
@@ -3,17 +3,51 @@
 
 from __future__ import annotations
```

Comment:
I assume this import isn't needed anymore?

Reply:
I'd have to check, might be for `-> "SomeClass":` underneath the SomeClass itself.